src/Jackett.Common/Definitions/hdcztorrent.yml

```diff
@@ -123,8 +123,8 @@ search:
     uploader: 0
     # 0 filename, 1 file & desc, 2 description
     options: 0
-    # 0 all, 1 oactive only, 2 dead only
-    active: 1
+    # 0 all, 1 active only, 2 dead only
+    active: 0
```

Comment:
oops, accidentally overrode the fix you snuck in while I was editing. fixed later with 7d9eea0